Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -792,20 +792,38 @@ void UpdateMenu()
 				else
 				{
 					IMenuItem menuItem = menu.Add(item.Text);
-					FileImageSource icon = item.Icon;
-					if (!string.IsNullOrEmpty(icon))
-					{
-						Drawable iconDrawable = context.GetFormsDrawable(icon);
-						if (iconDrawable != null)
-							menuItem.SetIcon(iconDrawable);
-					}
+					UpdateDrawableMenuItem(context, menuItem, item);
 					menuItem.SetEnabled(controller.IsEnabled);
 					menuItem.SetShowAsAction(ShowAsAction.Always);
 					menuItem.SetOnMenuItemClickListener(new GenericMenuClickListener(controller.Activate));
 				}
 			}
 		}
 
+		/// <summary>
+		/// For override specify tint color menu or specify drawable menu
+		/// Sample:
+		/// base.UpdateMenuItem(...)
+		/// menuItem.Icon.Mutate();
+		/// menuItem.Icon.SetColorFilter(this.Resources.GetColor([ResourceColor]), Android.Graphics.PorterDuff.Mode.SrcIn);
+		/// 
+		/// Sample with drawable badge count:
+		/// menuItem.SetIcon(SetDrawableIcon(menuItem.Icon)));
+		/// 
+		/// In this fact, toolBarItem is necessary for access to custom property like badge count
+		/// </summary>
+		/// <param name="menuItem"></param>
```

Comment:
Please don't add inline XML docs, they are (unfortunately) not allowed

Reply:
oki  i removed comment